packages/snap/src/encryption.ts

```diff
@@ -71,15 +71,15 @@ export function encryptZkCert(
 export function decryptMessageToObject(
   encryptedZkCert: EncryptedZkCert,
   privKey: string,
-): Record<string, unknown> {
+) {
   const decryptedMessage = decryptSafely({
     encryptedData: encryptedZkCert,
     privateKey: privKey,
   });
   // decryptSafely says it would return a string, but it actually returns what came out of JSON.parse().
   // (https://github.com/MetaMask/eth-sig-util/blob/10206bf2f16f0b47b1f2da9a9cfbb39c6a7a7800/src/encryption.ts#L234)
   // So we can cast it to ZkCertRegistered here.
-  const decrypted = decryptedMessage as unknown as Record<string, unknown>;
+  const decrypted = decryptedMessage as unknown as { zkCertStandard: string };
```

Comment:
This looks wrong. When decrypting a certificate it should have more fields than zkCertStandard.
I think `Record<string, unknown>` is fitting because the snap can not be sure if the encrypted message is a properly formatted certificate.